pom.xml

```diff
@@ -297,5 +298,49 @@
                 </plugins>
             </build>
         </profile>
+        <profile>
+            <id>docker:bash/arm64v8</id>
+            <activation>
+                <activeByDefault>false</activeByDefault>
+            </activation>
+            <properties>
+                <!--                DIGEST for arm64v8-->
+                <docker.base.image>eclipse-temurin:17.0.6_10-jre-jammy@sha256:0563afe1aa8e2bfbf68c8ef633d67839321b8bd0a8d163f0dff2dfedef04af29</docker.base.image>
+                <docker.image.tag>docker-bash-arm64v8</docker.image.tag>
```

Comment:
remove the `docker` prefix from the `docker.image.tag` i.e. `docker-bash-amd64` -> `bash-amd64`,  `docker-bash-arm64v8` -> `bash-arm64v8` as we already know these are docker images because we've used the `docker images` command to list them.